includes/class-content-toc.php

```diff
@@ -489,11 +489,10 @@ public function register_shortcake_ui() {
 						'type'        => 'text',
 						'placeholder' => $this->headers,
 						'description' => sprintf(
-							__( 'Comma separated list of HTML elements that are considered for building Content TOC. For example, default elements are: %1$s NOTE: DO NOT use %2$s to wrap an element, i.e. for example it should be simply %3$s and not %4$s. If no elements are specified, the default ones will be used instead: %1$s', 'hm-content-toc' ),
+							__( 'Comma separated list of HTML element names to generate TOC for. For example, default elements are: %1$s. NOTE: use %2$s, not %3$s.', 'hm-content-toc' ),
```

Comment:
This could do with a similar `translators:` comment as the one in `Admin::setup_plugin_option_settings()`.